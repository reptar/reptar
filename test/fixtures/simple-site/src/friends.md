---
title: Friends
slug: friends
date: 2016-04-19
---

I love my friends!

{% for key, friend in site.data.friends %}
  <p>Name: {{friend.name}}</p>
  <p>Birthday: {{friend.birthday}}</p>
{% endfor %}
