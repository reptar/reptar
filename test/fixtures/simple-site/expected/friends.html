<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />

  <title>My Site Title</title>
  <meta name="description" content="Your website's description goes here.
">
  <meta name="keywords" content="" />

  <meta name="HandheldFriendly" content="True" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />

  <link rel="shortcut icon" href="/favicon.ico">

  <link rel="stylesheet" type="text/css" href="/css/main-d8c1b45cda.css" />
</head>
<body class="post-template">

  <header class="main-header">
    <div class="vertical">
      <div class="main-header-content">
        <h1 class="page-title">
          <a href="/">My Site Title</a>
        </h1>
      </div>
    </div>
  </header>

  <main id="content" class="content" role="main">
    

<article class="post">

  <header class="post-header">
    <h1 class="post-title">Friends</h1>
  </header>

  <section class="post-content">
    <p>I love my friends!</p>
  <p>Name: Angelica</p>
  <p>Birthday: May 12, 1988</p>
  <p>Name: Chuckie</p>
  <p>Birthday: April 15, 1989</p>
  <p>Name: Tommy</p>
  <p>Birthday: June 11th, 1990</p>

  </section>

</article>



    
  </main>

  <footer class="site-footer clearfix">
    <p class="footer-description">
      &copy; My Site Title
    </p>
    <p class="footer-published">
      joyfully published by <a href="https://github.com/reptar/reptar">Reptar</a>
    </p>
  </footer>

  <script type="text/javascript" src="/js/main.js"></script>
</body>
</html>
